src/content/docs/en/reference/experimental-flags/raw-env-values.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Experimental raw environment variables values
+sidebar:
+  label: Raw environment variables
+i18nReady: true
+---
+
+import Since from '~/components/Since.astro'
+
+<p>
+
+**Type:** `boolean`<br />
+**Default:** `false`<br />
+<Since v="5.12.0" />
+</p>
+
+Astro allows you to configure a [type-safe schema for your environment variables](/en/guides/environment-variables/#type-safe-environment-variables), and converts variables imported via `astro:env` into the expected type.
+
+However, Astro also converts your environment variables used through `import.meta.env` in some cases, and this can prevent access to some values such as the strings `"true"` (which is converted to a boolean value), and `"1"` (which is converted to a number).
+
+The `experimental.rawEnvValues` flag disables coercion of `import.meta.env` values that are populated from `process.env`, allowing you to use the raw value.
+
+To disable Astro's coercion on values used through `import.meta.env`, set the `experimental.rawEnvValues` flag to `true` in your Astro configuration:
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from "astro/config"
+
+export default defineConfig({
+  experimental: {
+    rawEnvValues: true,
+  }
+})
+```
+
+## Usage
+
+Enabling this experimental flag will no longer convert string values into booleans or numbers. This aligns `import.meta.env`'s behavior in Astro with [Vite](https://vite.dev/guide/env-and-mode.html#env-variables).
+
+In a future major version, Astro will switch to not coercing `import.meta.env` values by default, but you can opt in to the future behavior early using the `experimental.rawEnvValues` flag and if necessary, [updating your project](#updating-your-project) accordingly.
+
+### Updating your project
+
+If you were relying on this coercion, you may need to update your project code to apply it manually:
```

Comment:
This example is great! Just checking though, you would *not* expect people to be updating any environment variables themselves? Are these things they would typically not have access to, so the real workaround is in fixing the logic to use the raw value?

Reply:
Yeah they have no control over the env var they receive, it always be a string. So the only way really is to update were it's handled in code